tests/test_isosurface.py

```
@@ -0,0 +1,546 @@
+    assert iso.color == "#FF0000"
+    assert iso.resolution == 0.5
+    assert iso.opacity == 0.8
+    assert iso.owner == "user123"
+
+
+def test_isosurface_frozen():
+    """Model is immutable (frozen=True)."""
+    from zndraw.geometries.isosurface import Isosurface
+
+    iso = Isosurface()
+    with pytest.raises(Exception):
```

Ruff (0.15.5)

[warning] Do not assert blind exception: `Exception`

(B017)